sumDouble/test/index.test.ts

```diff
@@ -0,0 +1,37 @@
+import { assert } from "chai";
+import { sumDouble } from "../src/index";
+
+
+/* 
+Given two int values, 
+  return their sum. 
+  
+Unless the two values
+  are the same, 
+    then return 
+      double their sum.
+
+
+sumDouble(1, 2) → 3
+sumDouble(3, 2) → 5
+sumDouble(2, 2) → 8
+*/
+
+describe("sumDouble", () => {
+  it("Should return 3 if num1 = 1 and num2 = 2.", () => {
```

Comment:
Test names should be phrased without having to use non-alphabetical characters, please do not use '=' signs and rather just be more verbose with your tests